src/pages/Main/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import CarouselContent from './components/CarouselContent'
+import Arrows from './components/Arrows'
+import Dots from './components/Dots'
+import { IMAGE_SLIDER } from './IMAGE_SLIDER'
+import './Carousel.scss'
+
+const Carousel = () => {
+  const lastIndex = IMAGE_SLIDER.length - 1
+  const [activeCarousel, setActiveCarousel] = useState(0)
+  return (
```

Comment:
```suggestion
  const [activeCarousel, setActiveCarousel] = useState(0)
  const lastIndex = IMAGE_SLIDER.length - 1

  return (
```
컨벤션은 아니지만 hook 을 사용하는 코드가 상단에 배치하는 것이 일반적인 방법이고
return 부분은 가독성을 위해 한칸 띄워주세요!